### PR TITLE
Remove dependency on 'pkg_resources' module

### PR DESCRIPTION
**Pull request by tlestang**
Instead specify location of data in a `DATA_PATH` module variable.

### CODE REVIEW

**Comment by tlestang** [comment]
Added a few commits to make it possible for the `readPlatformData` function to accept an optional `data_path` argument, setting the location of the coverage data. If not provided, default behavior is current behavio, i.e. look into the package's `data` directory.

this move https://github.com/NTD-Modelling-Consortium/endgame-project/issues/125 forward

File: tests/test_read_data.py [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    [2025.0, 0, 100, 0.85, 0, 2],
+]
+
+def test_read_platform_data():
+    result = readPlatformData("test_coverage_data.csv", "MDA")
+    assert result == EXPECTED
+
+def test_read_platform_data_other_data_path():
+    result = readPlatformData(
+        "test_coverage_data.csv",
```
Comment (thk123) [inline tests/test_read_data.py:22]: ⛏️ To make this test more robust, I would consider calling the file something besides the exact same name as is in the actual `data` directory, just to be sure it is loading the right file.

File: tests/test_read_data.py [hunk 2]
```diff
@@ -0,0 +1,34 @@
+import pytest
+
+from trachoma.trachoma_functions import readPlatformData
+
+EXPECTED = [
```
Comment (thk123) [inline tests/test_read_data.py:5]: 💡 In general I think it is best to inline this kind of thing to save readers jumping about (and maybe making the example even smaller so easier to check it is right. 

I really like this article on writing tests [why good developers write bad tests](https://mtlynch.io/good-developers-bad-tests/) which generally encourages copy-pasting things.

**Review by thk123 — APPROVED** [review]
Looks good!

File: tests/test_read_data.py (hunk 2 above)
Comment (tlestang) [reply tests/test_read_data.py:5]: Interesting blog post. It think I agree in general but just as I was removing this `EXPECTED` variable, I realised I would have to duplicate the expected list of lists as the expected returned value in both test functions. And with lots of number literals like that the risk of typos in the future is really high?

**Pull request merged**